### DISCUSSION TITLE
Kessel Leistung Hoch eingeschaltet

### DISCUSSION THREAD

**Discussion by FVerhamme** [dewenni/ESP_Buderus_KM271#89] [Q&A]
Hallo zusammen,

wenn ich ins Home Assistant Log schaue, dann sehe ich immer wieder die Einträge 'Kessel Leistung Hoch eingeschaltet'.
Zum gleichen Zeitpunkt steigt die Leistungsaufnahme von rund 8W auf ca. 54W an.
Es werden aber keine Pumpen (Zirkulation oder HK Pumpen) eingeschaltet. Auch die Brennersteuerung ist nicht an.

Könnt Ihr mir erklären, was zu diesem Zeitpunkt passiert? 

Ich kann das derzeit nur in Zusammen mit dem eingestellten Heizprogramm bringen. Dieser ist bei mir von 7:30-10:00 und von 16:00-22:00 Uhr.
Um Punkt 16:00 Uhr erschien die Meldung 'Kessel Leistung Hoch eingeschaltet'.
Seit diesem Zeitpunkt ist also der Verbrauch dauerhaft min. 54W! Die Heizung und Pumpen gehen natürlich jetzt Zwischendurch an, um Warmwasser zu machen oder zu Heizen. Aber danach fällt die Leistung nicht unter 54 Watt.

Gruß Frank

**Comment by the78mole** [comment]
Hallo Frank,

ich denke, die Frage kann Dir nur Buderus beantworten. Ich hätte fast auf die Ölvorheizung getippt, aber die sollte gefühlt etwas mehr Leistung aufnehmen (Ich hätte da so auf 200W getippt). Ist vielleicht die Ölpumpe an oder das Gebläse?

Grüße,
Daniel

**Reply by FVerhamme** [reply]
Hallo Daniel,
Beides ist nicht der Fall. Die laufen nur, wenn der Brenner tatsächlich anspringen soll.
Aber um Punkt 22:00 Uhr ist die Leistung dann wieder auf 6W gefallen und im Log erscheint "Kessel Leistung Hoch ausgeschaltet".
Es sieht also so aus, dass im Zeitraum des Heizprogramm (in meinem Fall 7-10 und 16-22 Uhr) die Kesselleistung hoch eingeschaltet wird und der Verbrauch h von ca. 6W auf ca. 54W ansteigt.
Interessant ist, dass ein den 30 Minuten vor dem Zeitraum, in denen sich die Heizung um Warmwasser kümmert, die Kesselleistung hoch nicht aktiviert wird.

Kann das jemand von Euch bestätigen?

**Reply by the78mole** [reply]
Hallo Frank,

das scheint sich niemand so wirklich auszukennen... Bei mir selbst ein Messgerät in die Zuleitung einzuschleifen, macht keinen Sinn. Meine Ölheizung macht nur noch sporadisch mal Warmwasser bzw. denkt sie, dass sie manchmal machen soll :-P

Grüße,
Daniel

**Reply by dewenni** [reply]
@FVerhamme ich habe gerade mal bei mir geschaut.
Der Status "Kessel Leistung hoch" ist bei den letzten 4 Brenner Starts nur ein mal kurz auf 0 gegangen. Ansonsten bleibt der auf 1

<img width="1640" alt="image" src="https://github.com/dewenni/ESP_Buderus_KM271/assets/46074831/5a0087b3-2ca8-4170-8c0a-b2aa4c415583">

Der Verbrauch geht bei mir auch immer auf die 6W zurück nachdem der Brenner aus geht.

**Comment by FVerhamme** [comment]
Danke,

ich werde dann mal Versuchen, mit Buderus in Kontakt zu treten um heraus zu bekommen, was hier eigentlich passiert und warum die Kesselleistung auf hoch geschaltet wird.

Gruß Frank

**Comment by DonCanDo** [comment]
Moin,

da bei mir der Wert Kessel Betrieb auch immer mit spielt, gehe ich davon aus, dass das Entweder ein Feedback vom Brennermodul zur Hauptplatine ist oder eine Bedingung die für das Brennermodul gesetzt wird. Mir kam mal die Idee dem Indiz Warmwasser nachzugehen, dass die Brennermodulation dadurch vielleicht zwischen WW und HK bei der Anforderung der Heizkurve unterscheidet aber in letzter Zeit läuft und saugt mein Trecker ganz fein, dass ich dem nicht weiter nach ging.

@dewenni @the78mole Hier mal noch mal die unendlich Dankbarkeit dass ich meine Warmwassertemperatur endlich unterschiedlich steuern kann. Der Warmwasservorrang hat mir immer jeden Tag meinen Ölbedarf versaut wenn ich das Warmwasser nicht brauche.
